### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
**PRT(Peer Review Template)**

- [o]  **1. 주어진 문제를 해결하는 완성된 코드가 제출되었나요? (완성도)**
    - 문제에서 요구하는 최종 결과물이 첨부되었는지 확인
    - 문제를 해결하는 완성된 코드란 프로젝트 루브릭 3개 중 2개, 
    퀘스트 문제 요구조건 등을 지칭
        - 해당 조건을 만족하는 부분의 코드 및 결과물을 캡쳐하여 사진으로 첨부
        - Project 1
        스크린샷 2024-05-22 17 36 07

        - Project 2
          스크린샷 2024-05-22 17 38 23
스크린샷 2024-05-22 17 49 39

- [o]  **2. 프로젝트에서 핵심적인 부분에 대한 설명이 주석(닥스트링) 및 마크다운 형태로 잘 기록되어있나요? (설명)**
    - [o]  모델 선정 이유
    - [o]  Metrics 선정 이유
    - [o]  Loss 선정 이유

- [o]  **3. 체크리스트에 해당하는 항목들을 모두 수행하였나요? (문제 해결)**
    - [o]  데이터를 분할하여 프로젝트를 진행했나요? (train, validation, test 데이터로 구분)
    - [o]  하이퍼파라미터를 변경해가며 여러 시도를 했나요? (learning rate, dropout rate, unit, batch size, epoch 등)
    - [o]  각 실험을 시각화하여 비교하였나요?
    - [o]  모든 실험 결과가 기록되었나요?

- [o]  **4. 프로젝트에 대한 회고가 상세히 기록 되어 있나요? (회고, 정리)**
    - [o]  배운 점
      -현동님에 비해 저는 기존에 명시되었던 파라미터만 사용해봐서 추이를 보기 어려웠기 때문에 최적의 하이퍼파라미터를 찾았다고 보기 어렵습니다. 
    - [ ]  아쉬운 점
    - [o]  느낀 점
      - 현동님께서는 첫번째 프로젝트에서 하이퍼 파라미터 따라서 결과가 어떻게 변화하는지 보고싶다고 하셔서 여러 번 튜닝을 거쳐서 최적의 하이퍼 파라미터를 찾아내시는 과정을 본 받고싶었습니다. 수고하셨습니다. 오늘 처럼만 잘 기록하시면 일취월장 하실 것 같아요.
    - github 너무 깔끔하게 폴더 정리 해두셔서 보기 좋았습니다. ☺️
    - [ ]  어려웠던 점

### CODE REVIEW

**Comment by DevHDL** [comment]
@JeenaKim99 pull request 하기 전에 제가 다른 commit을 해서 현재 충돌이 났네요.
올려주신 내용 토대로 제 local git에서 새로 올리도록 하겠습니다.
고생하셨습니다.
